src/styles/dynamaic-variables.scss

```diff
@@ -70,10 +70,10 @@
   --playkit-tone-1-color: #ffffff;
```

Comment:
@lianbenjamin Check the values that are defined by hue-saturation. It looks like their values were also updated.

Reply:
@SivanA-Kaltura i've checked them with an online hsl to hex tool (and vice-versa)- they match the design.